packages/gatsby/src/commands/develop.ts

```diff
@@ -363,9 +363,7 @@ module.exports = async (program: IProgram): Promise<void> => {
 
   if (process.env.GATSBY_EXPERIMENTAL_PAGE_BUILD_ON_DATA_CHANGES) {
     report.panic(
-      `The flag ${chalk.yellow(
-        `GATSBY_EXPERIMENTAL_PAGE_BUILD_ON_DATA_CHANGES`
-      )} is not available with ${chalk.cyan(
+      `Conditional page builds are not available with ${chalk.cyan(
```

Comment:
we should keep some kind of reference to `GATSBY_EXPERIMENTAL_PAGE_BUILD_ON_DATA_CHANGES` flag name in message, so this message is still actionable for users that don't know that "Conditional page builds" is human readable name for the flag.

I'm bad at copy writing so can't suggest nice message, but I would like both "Conditional page builds" and "GATSBY_EXPERIMENTAL_PAGE_BUILD_ON_DATA_CHANGES" mentioned here.

Maybe even add link to docs as we do have note there:

> This feature is not available with `gatsby develop`.

But I always feel uneasy about adding links in error messages as content might move (not huge problem with redirects in place) or content can change, so use your judgement on that one :)

Reply:
I actually probably would keep this as it was -- there wasn't a reference to incremental builds here?

Reply:
I've reverted this